.travis.yml

```diff
@@ -165,24 +166,24 @@ install:
       if [[ ! $skip && $deps ]]; then mv composer.json.phpunit composer.json; fi
 
       if [[ ! $skip && $PHP = 7.* ]]; then
-          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; composer require --dev --no-update mongodb/mongodb)
+          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; composer require --dev --no-update mongodb/mongodb || true)
       fi
 
-    - if [[ ! $skip ]]; then $COMPOSER_UP; fi
-    - if [[ ! $skip ]]; then ./phpunit install; fi
+    - if [[ ! $skip ]]; then $COMPOSER_UP || true; fi
+    - if [[ ! $skip ]]; then ./phpunit install || true; fi
```

Comment:
Adding `|| true` is totally a no-go. If the setup of the testsuite fails, we don't want to ignore the failure and try the next step. This would just lead to an output impossible to understand due to weird failures later (trying to use the packages which were not installed properly before)

Reply:
Do you know how we could debug this segmentation fault ? I think something needs to be done globally in the script since it does not work event piping true